classes/controller/class-ensapi-controller.php

```diff
@@ -128,35 +128,37 @@ public function get_pages( $params = [
 	] ) {
 
 		$response['body'] = get_transient( 'ens_pages_response_' . implode( '_', $params ) );
-		if ( ! $response['body'] ) {
-			$url = add_query_arg(
-				[
-					'type'   => strtolower( $params['type'] ),
-					'status' => $params['status'],
-				],
-				self::ENS_PAGES_URL
-			);
+		if ( $response['body'] ) {
+			return json_decode( $response['body'], true );
```

Comment:
Since we're already changing this part, now we can make this a separate variable as it's either returned immediately, or would be overwritten [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/308/files#diff-9fe83910b67fb870e125aead11d0e5d2R144). It's easier to follow the code if these are separate, so we could make it something like 
```
$from_cache= get_transient( 'ens_pages_response_' . implode( '_', $params ) );
if ( $from_cache ) {
	return json_decode( $from_cache, true );
}
```